lib/stdlib/src/json.erl

```diff
@@ -712,14 +712,14 @@ format_object([[_Comma,KeyIndent|Entry]], Indent) ->
     {_, Rest} = string:take(Value, [$\s,$\n]),
     [CP|_] = string:next_codepoint(Rest),
     if CP =:= ${ ->
-            ["{", KeyIndent, Entry, Indent, "}"];
+            [${, KeyIndent, Entry, Indent, $}];
        CP =:= $[ ->
-            ["{", KeyIndent, Entry, Indent, "}"];
+            [${, KeyIndent, Entry, Indent, $}];
        true ->
             ["{ ", Entry, " }"]
```

Comment:
Would this change also improve performance?

```suggestion
            [${, $\s, Entry, $\s, $}]
```

Reply:
I think it makes sense, @sabiwara can you fix that as well.

Reply:
I haven't benchmarked this one so I'm not sure, but unlike `${` which has no downside, for`${, $\s` I'd expect to have an extra cost of generating slightly bigger lists, so I thought it might not be such a clear win?

Reply:
${, $\s should be slightly smaller it still avoids creating another nested list.